app/views/redmine_required_field/_settings.html.erb

```diff
@@ -43,36 +44,43 @@
     </legend>
 
     <div id='rrf_issue_required_form' class='tabular settings'>
-
-      <p>
+      <div style="display:flex; width:100%;" >
+      <p style="padding: 10px; display:flex;flex-direction: column;">
         <%= content_tag :label, t(:label_project_plural) %>
         <%= select_tag 'rrf_responsive_setting[project_ids]', options_for_select(rrf_all_projects), :multiple => true, :size => 30 %>
       </p>
 
-      <p>
+      <p style="padding: 10px; display:flex;flex-direction: column;">
         <%= content_tag :label, t(:label_tracker_plural) %>
         <%= select_tag 'rrf_responsive_setting[tracker_ids]', options_for_select(rrf_all_trackers), :multiple => true, :size => 30 %>
       </p>
 
-      <p>
+      <p style="padding: 10px; display:flex;flex-direction: column;">
         <%= content_tag :label, t(:label_issue_status_plural) %>
         <%= select_tag 'rrf_responsive_setting[issue_status_ids]', options_for_select(rrf_all_issue_statuses), :multiple => true, :size => 30 %>
       </p>
-
+      <p style="padding: 10px; display:flex;flex-direction: column;">
+        <%= content_tag :label, t(:label_role_plural) %>
+        <%= select_tag 'rrf_responsive_setting[role_ids]', options_for_select(rrf_all_roles), :multiple => true, :size => 30 %>
+      </p>
+      </div>
+      <%= check_box_tag 'check_assignee_only' %>
+      <%= label_tag 'check_assignee_only', l('redmine_required_field.settings.check_assignee_only'),class:'inline' %>
+      <br>
+      <%= check_box_tag 'role_excluded' %>
+      <%= label_tag 'role_excluded', l('redmine_required_field.settings.exclude_roles'),class:'inline' %>
+      
       <p>
         <%= submit_tag l(:button_add), :name => :responsive_update, :id => :responsive_update %>
         (**)
       </p>
 
-
       <br>
       <span>(*) = <%= l('redmine_required_field.closed_status') %></span>
       <br>
       <span>(**) = <%= l('redmine_required_field.need_to_click_apply_to_save_settings') %></span>
       <br>
 
-     </div>
```

Comment:
doesn't removing this end of div line break the layout ?

Reply:
i closed the div in line 66, so the layout is not broken. this was done to structure the checkboxes for the conditional options under the selection inputs